classes/renderer/hasmany.php

```diff
@@ -235,7 +235,7 @@ public function before_save($item, $data)
 
             // Trigger the before save of all the fields of the has_many
             $config = static::getConfig($relatedItem, array());
-            $fieldset = static::getFieldSet($config, $relatedItem);
+            $fieldset = static::getFieldSet($config, $relatedItem, $this->renderer_options);
```

Comment:
this is not needed, the signature is `protected static function getFieldSet($config, $item)`